.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: nix
 env:
-  - NIXPKGS_CHANNEL=17.09
-  - NIXPKGS_CHANNEL=unstable
+  - NIXPKGS_CHANNEL=nixos-17.09
```

Comment:
Maybe also bump the stable version to 18.09?

Reply:
maybe, I like that yarn2nix is back-compat with at least 2 versions

Reply:
did a compromise in 
62b3418